src/gui/README.md

```diff
@@ -2199,3 +2141,61 @@ Result:
     "47.88.33.156:6000"
 ]
 ```
+
+### Status report for default connection peers
+
+```
+URI: /network/connections/default
+
+Method: GET
+```
+
+Example:
+
+```sh
+curl http://127.0.0.1:6420/network/connections/default
+```
+
+Response:
+
+```json
+{
+  "count": 8,
```

Comment:
Wrong indentation.

You can get an example of a valid response with correct formatting by making the api call to a running node